### PR TITLE
Update foreman_expire_hosts to 6.0.1

### PR DESCRIPTION
**Pull request by timogoebel**
(cherry picked from commit 1ff39ef1431700b56fbbf6fc766dfd471756f9ae)

### CODE REVIEW

**Comment by evgeni** [comment]
When updating to 6.0.0 in rpm/develop (32d9126f553898d2a60512c6f05ebb73580172b2) you bumped foreman_min_version to 1.18.0, this is not CP'ed. intentionally? Generally I'd prefer if the history in develop and in stable branches is "identical" (i.e. not skipping the 6.0.0 commit)

**Comment by timogoebel** [comment]
@evgeni: I added the commit to the history.

File: packages/plugins/rubygem-foreman_expire_hosts/rubygem-foreman_expire_hosts.spec [hunk 1]
```diff
@@ -116,6 +116,13 @@ install -Dp -m0644 %{SOURCE1} %{buildroot}%{?scl:%_root_sysconfdir}%{!?scl:%_sys
 exit 0
 
 %changelog
+=======
```
Comment (evgeni) [inline packages/plugins/rubygem-foreman_expire_hosts/rubygem-foreman_expire_hosts.spec]: toot toot, conflict train ;)

Comment (timogoebel) [reply packages/plugins/rubygem-foreman_expire_hosts/rubygem-foreman_expire_hosts.spec]: Fixed, thanks. 😊

File: packages/plugins/rubygem-foreman_expire_hosts/rubygem-foreman_expire_hosts.spec [hunk 2]
```diff
@@ -4,12 +4,12 @@
...
 %global gem_name foreman_expire_hosts
 %global plugin_name expire_hosts
-%global foreman_min_version 1.17.0
+%global foreman_min_version 1.18.0
 
 Summary:    Foreman plugin for limiting host lifetime
 Name:       %{?scl_prefix}rubygem-%{gem_name}
-Version:    5.1.0
-Release:    4%{?foremandist}%{?dist}
+Version:    6.0.0
```
Comment (evgeni) [inline packages/plugins/rubygem-foreman_expire_hosts/rubygem-foreman_expire_hosts.spec]:
```suggestion
Version:    6.0.1
```

Comment (timogoebel) [reply packages/plugins/rubygem-foreman_expire_hosts/rubygem-foreman_expire_hosts.spec]: 🚂, fixed.

**Pull request merged**